src/variant.cpp

```diff
@@ -1775,6 +1800,8 @@ void VariantMap::init() {
     add("isolation", isolation_variant());
     add("isolation7x7", isolation7x7_variant());
     add("snailtrail", snailtrail_variant());
+    add("fox_and_hounds", fox_and_hounds_variant());
```

Comment:
So far underscores aren't used in official variant names in Fairy-SF, so I would suggest to use either a dash or no separator. There isn't any specific reason to avoid underscores, e.g., there isn't a technical limitation, it is just about the existing (unwritten) naming convention.

Reply:
Done.